### ISSUE TITLE
 [Screen Reader - PowerApps Admin - Select organization] In scan mode Narrator focus moves out of 'Saving connection set failed" pop up window when press Up arrow keys 

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7480]
<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
  
Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
**Component: Modal**
**Steps:**
Please go to the office fabric page for Modal (https://developer.microsoft.com/en-us/fabric#/components/modal)
open the Modal
Turn the narrator scan mode on.
Pressing down arrow key, focus does not go out of the modal.
but pressing up arrow key, focus moves out of the modal. (see the screenshot as well).

**### Environment Information** 
OS version: Version 1803 (OS build 17134.167)
Browser: Edge, Chrome
Screen Reader: Narrator, JAWS

- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

### Please provide a reproduction of the bug in a codepen:
 
**Pre-Requisite:**
1. Turn on narrator by pressing Ctrl + window + ENTER combination.

**Repro Steps:**
1. Navigate to https://admin.powerplatform.microsoft.com/ext/DataIntegration and login with intusr.
2. Navigate to the Data Integration menu item on the left bar and select it.
3. Navigate to "+New Connection" on the top right corner and select it.
4. Connection Set name pop up appears, enter the connection name and select next button.
5. First app connection info screen appears, fill in the connection and environment and select next button.
6. Second app connection info screen appears, fill in the connection and environment and select next button.
7. Select organization screen appears, navigate on the combo box and select any value.
8. Navigate to save button and select it. Connection gets saved.
9. Navigate with the Narrator in scan on the 'Saving connection set failed' pop up window. 
10. Verify whether the focus of narrator in scan mode moves out of 'Saving connection set failed pop up window or not.
 
#### Actual behavior:
 In scan mode Narrator focus moves out of 'Saving connection set failed" pop up window when press Up arrow keys 

 
#### Expected behavior:
 In scan mode Narrator focus moves out of 'Saving connection set failed" pop up window when press Up arrow keys. when user press Up arrow key, focus should not moves out of the dialog, it should navigate inside the pop up. 
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low)  Blocking
 
Products/sites affected: (if applicable)
![focus moves out of the dialog via up arrow key](https://user-images.githubusercontent.com/30456910/50477241-77038b00-09f1-11e9-8233-3627c78d2d92.PNG)

**Comment by micahgodbolt** [comment]
This is a duplicate of #6422. We have gotten our modal to 'spec', but narrator does not support aria-modal=true yet. This support is in their backlog, and once the work is done, this will work in scan mode.

**Comment by sonimishra** [comment]
As checked, this issue is marked duplicated with https://github.com/OfficeDev/office-ui-fabric-react/pull/6884 .
